gcloud/streaming/http_wrapper.py

```diff
@@ -340,7 +340,7 @@ def handle_http_exceptions(retry_args):
         logging.debug('Response returned a retry-after header, retrying')
         retry_after = retry_args.exc.retry_after
     else:
-        raise
+        raise  # pylint: disable=misplaced-bare-raise
```

Comment:
@tseaver Shall we file a bug to get this bare raise out of here? ISTM we can just add an argument to this function which accepts a caught exception and then it can just

```python
raise caught_exc
```